### DISCUSSION TITLE
How to change the homepage content?

### DISCUSSION THREAD

**Discussion by SWHL** [colinwilson/lotusdocs#54] [Q&A] [answered]
Thanks for your work.
I like the theme very much, and I have deployed the documentation site on my projects.
But I want to change the default content on the homepage, I can't find where to change it.

The default content is like this [link](https://rapidai.github.io/LabelConvert/).  
I want to change the content to  be related to my project. 

Appreciate any help on this.

**Comment by colinwilson** [comment, marked as answer]
Create / Update your own `landing.yaml` data config for your site:

```treeview
data/
└── landing.yaml
```
This will override the theme's default [`landing.yaml`](https://github.com/colinwilson/lotusdocs/blob/release/data/landing.yaml) config. <-- You can use this config as a guide.

I've almost finished the documentation for the managing the theme's landing page. It'll be part of the first release.
